docs/semver.md

```diff
@@ -19,7 +19,9 @@ versions:
   ([`general.data_directory`](shadow_config_spec.md#generaldata_directory))
   will not change.
 - The PID numbering (the values and their order) will not change.
-- Any [supported platforms](supported_platforms.md) will not be dropped.
+- Any [supported platforms](supported_platforms.md) will not be dropped, unless
+  those platforms are considered "unsupported" and no longer receive free
```

Comment:
> Maybe something like "unless the developer of the platform itself (e.g. Canonical Ltd, The Debian Project, or The Fedora Project) has ceased free updates and support of that platform.

I think this means that we'd only support Debian for 3 years then, since years 3-5 are only community supported and not "The Debian Project" supported.

Reply:
I changed it to the following, lmk what you think:

Support for any of Shadow's supported platforms will not be dropped, unless those platforms no longer receive free updates and support from the distribution's developer or the community.

Reply:
I dropped the "or the community".